pipeline/acquisition.py

```diff
@@ -0,0 +1,390 @@
+        return None
+
+
+@schema
+class UnitSpikeTimes(dj.Imported):
+    definition = """ 
+    -> ProbeInsertion
+    unit_id : smallint
+    ---
+    -> reference.Probe.Channel
+    spike_times: longblob # time of each spike, with respect to the start of session 
+    unit_cell_type: varchar(32) # e.g. cell-type of this unit (e.g. wide width, narrow width spiking)
+    unit_depth_x: float
+    unit_depth_y: float
+    unit_depth_z: float
+    spike_waveform: longblob # waveform(s) of each spike at each spike time (spike_time x waveform_timestamps)
+    """
+        
+    def make(self,key):
+        ############## Dataset #################
+        sess_data_dir = os.path.join('..','data','extracellular','datafiles')
+                
+        # Get the Session definition from the keys of this session
+        animal_id = key['subject_id']
+        date_of_experiment = key['session_time']
+        
+        # Search the files in filenames to find a match for "this" session (based on key)
+        sess_data_file = utilities.find_session_matched_nwbfile(sess_data_dir, animal_id, date_of_experiment)
+        if sess_data_file is None: 
+            return
+        nwb = h5.File(os.path.join(sess_data_dir,sess_data_file), 'r')
+
+        # ------ Spike ------
+        ec_event_waveform = nwb['processing']['extracellular_units']['EventWaveform']
+        ec_unit_times = nwb['processing']['extracellular_units']['UnitTimes']
+        # - unit cell type
+        cell_type = {}
+        for tmp_str in ec_unit_times.get('cell_types').value:
+            tmp_str = tmp_str.decode('UTF-8')
+            split_str = re.split(' - ',tmp_str)
+            cell_type[split_str[0]] = split_str[1]
+        # - unit info
+        print('Inserting spike unit: ', end="")
+        for unit_str in ec_event_waveform.keys():
+            unit_id = int(re.search('\d+',unit_str).group())
+            unit_depth = ec_unit_times.get(unit_str).get('depth').value
+            key['unit_id'] = unit_id
+            key['channel_id'] = ec_event_waveform.get(unit_str).get('electrode_idx').value.item(0) - 1  # TODO: check if electrode_idx has MATLAB idx (starts at 1)
+            key['spike_times'] = ec_unit_times.get(unit_str).get('times').value
+            key['unit_cell_type'] = cell_type[unit_str]
+            key['unit_depth_x'] = unit_depth[0]
+            key['unit_depth_y'] = unit_depth[1]
+            key['unit_depth_z'] = unit_depth[2]
+            key['spike_waveform'] = ec_event_waveform.get(unit_str).get('data').value
+            self.insert1(key, ignore_extra_fields=True)
+            print(f'{unit_id} ',end="")
+        print('')
+        nwb.close()
+    
+
+@schema
+class TrialSet(dj.Imported):
+    definition = """
+    -> Session
+    ---
+    trial_counts: int # total number of trials
+    """
+    
+    class Trial(dj.Part):
+        definition = """
+        -> master
+        trial_id: smallint           # id of this trial in this trial set
```

Comment:
Overall, I think we can get away to adding `_id` to the primary key attributes.  We can just use words `trial`, `unit`, etc.